modules/hooks.nix

```diff
@@ -565,6 +565,20 @@ in
           };
         };
       };
+      golines = mkOption {
+        description = "golines hook";
+        type = types.submodule {
+          imports = [ hookModule ];
+          options.settings = {
+            flags = mkOption {
```

Comment:
For what it's worth, all hooks accept an `args` option that does this.

Reply:
ahh good to know for the next one thanks